src/MongoDB.Driver.Core/Core/Misc/Feature.cs

```diff
@@ -75,6 +75,7 @@ public class Feature
         private static readonly Feature __scramSha256Authentication = new Feature("ScramSha256Authentication", new SemanticVersion(4, 0, 0, ""));
         private static readonly Feature __serverExtractsUsernameFromX509Certificate = new Feature("ServerExtractsUsernameFromX509Certificate", new SemanticVersion(3, 3, 12));
         private static readonly Feature __shardedTransactions = new Feature("ShardedTransactions", new SemanticVersion(4, 1, 6));
+        private static readonly Feature __tailableCursor = new Feature("TailableCursor", new SemanticVersion(3, 2, 0));
```

Comment:
As I see from this link:https://docs.mongodb.com/manual/reference/method/cursor.tailable/ and the local tests, we don't support a tailable cursor before the servers less than `3.2`